src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -267,18 +261,6 @@
   transform: var(--pf-c-app-launcher__menu-item-external-icon--Transform); // move icon a bit to appear more visually centered
 }
 
-.pf-c-app-launcher__separator {
-  height: var(--pf-c-app-launcher__separator--Height);
-  margin-top: var(--pf-c-app-launcher__separator--MarginTop);
-  margin-bottom: var(--pf-c-app-launcher__separator--MarginBottom);
-  background-color: var(--pf-c-app-launcher__separator--BackgroundColor);
-  border: 0; // for use when separator is an <hr>
-
-  &:last-child {
```

Comment:
Can you add this back in? And maybe leave a comment for why it exists since it's not a normal use case. It's to support adding a divider as the last item of a group as a means to separate groups. This was requested by an application due to their loop logic, and it's easy enough to support.

The space should look the same whether the divider is between sections or as the last list item in a section.